targets/trusted_root.json

```diff
@@ -76,12 +76,34 @@
         "rawBytes": "MFkwEwYHKoZIzj0CAQYIKoZIzj0DAQcDQgAE8gEDKNme8AnXuPBgHjrtXdS6miHqc24CRblNEOFpiJRngeq8Ko73Y+K18yRYVf1DXD4AVLwvKyzdNdl5n0jUSQ==",
         "keyDetails": "PKIX_ECDSA_P256_SHA_256",
         "validFor": {
-          "start": "2022-07-01T00:00:00.000Z"
+          "start": "2022-07-01T00:00:00Z"
         }
       },
       "logId": {
         "keyId": "KzC83GiIyeLh2CYpXnQfSDkxlgLynDPLXkNA/rKshno="
       }
     }
+  ],
+  "timestampAuthorities": [
+    {
+      "subject": {
+        "organization": "sigstore.dev",
```

Comment:
I wonder if we should reissue the root so that organization is sigstage.dev, just so it's more obvious this is for staging? This would align with fulcio

Reply:
I left it as is when I worked on timestamp-authority because:
* fulcio staging issuer is also "Issuer: O=sigstore.dev, CN=sigstore"
* it's currently hard coded in the code so would be a little more work

in hindsight this issue would have been more obvious if the issuer was different